app.py

```diff
@@ -1,31 +1,307 @@
-import pymongo
 import os
-from flask import Flask
+from flask import Flask, render_template, redirect, request, url_for, jsonify, flash
+from flask_pymongo import PyMongo, pymongo
+from bson.objectid import ObjectId
+from bson import SON
+from pprint import pprint
+from pymongo import ASCENDING, DESCENDING, TEXT
+from functions import get_pages, generate_pagination_links, get_countries
+from flask_wtf import FlaskForm, Form
+from wtforms import  TextField, SelectField, TextAreaField, validators, StringField, SubmitField
 
 app = Flask(__name__)
+app.secret_key = 'some_secret'
```

Comment:
I know this works fine but it is not the best secret string to use... search google for a secret string generator and generate a better string.

Reply:
Thanks for that I've updated this now!